mlrun/api/api/endpoints/functions.py

```diff
@@ -223,8 +256,11 @@ def _build_function(db_session, function, with_mlrun):
         run_db = get_run_db_instance(db_session)
         fn.set_db_connection(run_db)
         fn.save(versioned=False)
-
-        ready = build_runtime(fn, with_mlrun)
+        if fn.kind in RuntimeKinds.serverless_runtimes():
+            deploy_nuclio_function(fn)
+            ready = False
```

Comment:
Add a comment explaining why it's not ready (IIUC - because this is just triggering the deployment but not waiting for it to finish, I would also add a TODO that other runtimes deployment should be changed to act this way as well

Reply:
done